publications/2024_jaszowiec/poster.tex

```diff
@@ -0,0 +1,180 @@
+        \textbf{DMD (digital micromirror device) chip}\cite{dlp}. We use
+        combination of white and black pixels in the image to select which part
+        of the wafer should be exposed to UV. There is a 3D-printed shutter
+        connected to the electric motor driven by Raspberry Pi Pico
+        W\cite{picow}, which can be used to expose the wafer for the desired
+        duration, controlling the amount of UV that hits the photoresist.  We
+        provide the microcontroller \textbf{firmware as an open source
+        software}\cite{firmware}.
+        \begin{tikzfigure}
+            \includegraphics[width=0.45\textwidth,height=0.15\textwidth]{media/pl_background.jpg}
+        \end{tikzfigure}
+    }
+
+    \block{Contacts}{
+        Contacts are done using silver epoxy and copper wires. The other end of
+        the wire can then make a direct contact with the probe or soldered into
+        a prototyping board or PCB for testing.
+    }
+
+    \column{0.50}
+    \block{Hotplate}{
+        \begin{wrapfigure}{r}{0.2\textwidth}
+            \begin{tikzfigure}
+                \includegraphics[width=400pt]{media/hotplate_noplate.jpg}
+            \end{tikzfigure}
+        \end{wrapfigure}
+        Electric hotplate that can reach up to \textbf{400°C}. Based on the
+        firebricks and resistance wire. Used for soft and post-exposure bake of
+        photoresist, synthesis of tetraethyl orthosilicate based Spin-On-Dopant
+        and it's predeposition step before diffusion doping in the furnace.
+        Raspberry Pi Pico W microcontroller\cite{picow} is used for wireless
+        access, controlling rate of heating and automatic temperature steering.
+        We provide the microcontroller \textbf{firmware as an open source
+        software}\cite{firmware}.
+    }
+
+    \block{Doping}{
+        \textbf{We propose a modified process to the one described in Fangsuwannarak et
+        Al. 2019 \cite{Fangsuwannarak}}. It yields a less, but sufficiently viscous
+        dopant and significantly \textbf{extends it's shelf life in room temperature}.
+        We spin coat the dopant on the wafer, do pre deposition on hot plate
+        and thermal diffusion in the furnace. This creates
+        N-doped wells and PSG. The PSG is stripped using \( NH_{4}F\cdot HF\)
+        solution dissolved in DI water.
+        For \textbf{selective doping} we use thick (\(\sim\)500nm) thermally grown
+        \(SiO_{2}\), that stops spin coated SOD from penetrating too far into
+        the wafer.
+        We control for rate of doping by doing a \textbf{double-oxidation}
+        process, which grows two layers of \(SiO_{2}\), one, for completely
+        stopping the doping \(\sim\)500nm and another in a range of 30-100nm,
+        which acts as a doping limiter and not a complete stopper.
+    }
+
+    \block{Spin coating}{
+        We successfully used a 3000 RPM, 12V \textbf{computer fan} to
```

Comment:
idk if this fan had 3000RPM after we removed the wings, i suppose that value has been bigger

Reply:
We don't say it was turning at that rate, because we simply don't know - we never measured it. We just know that it was 3000RPM computer fan.